src/main/scala/sbtghactions/UseRef.scala

```diff
@@ -19,7 +19,7 @@ package sbtghactions
 sealed trait UseRef extends Product with Serializable
 
 object UseRef {
-  final case class Public(owner: String, repo: String, ref: String) extends UseRef
+  final case class Public(owner: String, repo: String, rev: String, ref: String) extends UseRef
```

Comment:
Seeing as `ref` is now always rendered as a comment, I think it makes to keep `ref` as the actual reference used (be it a hash, tag, or branch name), and instead introduce `comment: Option[String] = None`. This way we'd retain source compatibility, also allowing users to opt-in to reference actions by a hash + comment, if they aren't already.

Reply:
I'm not sure - I think it's actually a feature that users would have to review the call site to make sure the third argument is a `rev`. I could see making the fourth parameter a `comment: Option[String]` though, no strong preference.